contracts/dao/voting/lockdrop-vault/src/tests.rs

```diff
@@ -810,7 +810,7 @@ fn test_total_power_at_height() {
     );
 
     let resp = get_total_power_at_height(&mut app, addr, None);
-    assert_eq!(resp.power, Uint128::from(14u128));
+    assert_eq!(resp.power, Uint128::from(2121320u128 + 2828427u128)); // (3b / sqrt(2)) + (4b / sqrt(2))
```

Comment:
Let's make those values ("3b", "2" and so on) public constants. It's really hard to search for those values in the contracts' mock code and it works only if one deeply understands where to find them.

Reply:
And let's change 2 to 4 or something that has better sqrt for prettier numbers here

Reply:
done